src/mixins/filing-template-mixin.ts

```diff
@@ -1467,15 +1467,18 @@ export default class FilingTemplateMixin extends Mixins(AmalgamationMixin, DateM
   }
 
   /**
-   * Fixes addresses by deleting the type in case it is null. Also fixes null officer email.
-   * @param orgPeople The array of orgs/people
-   * @returns the array of orgs/people with fixed address types and email
+   * For all orgs/people elements, fixes the following properties:
+   * - delivery address type if it's null
+   * - mailing address type if it's null
+   * - officer email if it's null or empty
+   * @param orgPeople the original array of orgs/people
+   * @returns a new array of orgs/people with fixed properties
    */
-  fixNullAddressType (orgPeople: OrgPersonIF[]): OrgPersonIF[] {
+  fixOrgPeopleProperties (orgPeople: OrgPersonIF[]): OrgPersonIF[] {
     return orgPeople.map(p => {
       if (p.deliveryAddress?.addressType === null) delete p.deliveryAddress.addressType
       if (p.mailingAddress?.addressType === null) delete p.mailingAddress.addressType
-      if (p.officer?.email === null) delete p.officer.email
+      if (p.officer?.email === null || p.officer?.email === '') delete p.officer.email
```

Comment:
Even though the [JSON schema reference](https://json-schema.org/understanding-json-schema/reference/string) says that an empty ("") string is valid, it leaves room for some implementations to do things a bit differently. Apparently our BE implementation is different, so empty email addresses need to be deleted from the filing object/JSON.